CONTRIBUTING.md

```diff
@@ -1,3 +1,37 @@
+# CONTRIBUTING.md
+
+## What Cannot Be Included
+
+- TTS Staff should not include information that shouldn't be public.
+  - We already get training on this, but here are a few reminders about things we shouldn't include here:
+  - [Sensitive information, as described in our Open Source Policy practices guide](https://github.com/18F/open-source-policy/blob/master/practice.md#protecting-sensitive-information)
+  - Comments that can be easily interpreted as [endorsements](https://www.oge.gov/web/oge.nsf/Use%20of%20Government%20Position%20and%20Resources/17593AE8B3A597C685257E96006364E4?opendocument) (or other potential ethical issues)
+- Information that can easily go out of date and is already published somewhere else in a useful format. This includes:
+  - People's phone numbers, even if public information (such as their GSA work number). Link to the public [GSA staff directory](http://www.gsa.gov/portal/staffDirectory/searchStaffDirectory) or the access-controlled 18F contact spreadsheet instead of including phone numbers in the handbook
+  - Information that is already well-covered on GSA InSite - link there instead of reproducing it
+  - Specific TTS org chart info (such as lists of names of supervisors and facilitators). Link to the [TTS org chart](https://handbook.18f.gov/tts-org-chart/), the internal [18F org chart](https://handbook.18f.gov/org-chart/), or the internal [OPP org chart](https://handbook.18f.gov/opp-org-chart/) instead
+
+## A Few Things to Remember
+
+- This handbook website and repository are public
+- We're careful about publishing [information collected during research](https://handbook.18f.gov/research-guidelines/); [learn more](https://docs.google.com/document/d/1Xp4LxbW6cx61rXrsnnfIPCz6cglovHzZeEjCcnpIeaM/edit) and ask [#research](https://gsa-tts.slack.com/archives/research) for guidance first
+- Avoid "click here" links
```

Comment:
We should mention they're problematic for a11y and maybe provide links. There's some guidance from the w3c, for example, and maybe other good examples.